rest-tests/src/test/resources/features/common.feature

```diff
@@ -1,7 +1,32 @@
 Feature: Common scenarios
 
-@clean-deploy
-Scenario: Clean & deploy
-	Given clean default namespace
-	When deploy Syndesis from template
-	Then wait for Syndesis to become ready
\ No newline at end of file
+  @clean-deploy
+  Scenario: Clean & deploy
+    Given clean default namespace
+    When deploy Syndesis from template
+    Then wait for Syndesis to become ready
+
+  @integrations-twitter-to-salesforce
+  Scenario: Tw - sf integration
+    Given user cleans tw to sf scenario
+    And user creates the tw connection using "twitter_listen" template
+    And user creates sf connection
+    And user creates tw mention step with "twitter-mention-connector" action
+    And user creates basic tw to sf filter step
+    And user creates tw to sf mapper step
+    And user creates sf step for tw sf test
+    And user creates tw to sf integration
```

Comment:
I'd suggest to change this a bit a split this to 
```
When user creates tw to sf integration
Then wait for integration "<IntegrationName>" to be "Active"
```
wdyt?

Reply:
ok, sounds reasonable